app/data/models/claim.js

```diff
@@ -0,0 +1,30 @@
+module.exports = (sequelize, DataTypes) => {
+  const claim = sequelize.define('claim', {
+    calculationId: { type: DataTypes.INTEGER, primaryKey: true, autoIncrement: true },
+    paymentRequestId: DataTypes.INTEGER,
+    sbi: DataTypes.INTEGER,
```

Comment:
We may need `frn` here too as a `BIGINT`.  All payments are FRN focused so _if_ it's possible to get from DWH, it might simplify matching.

Happy it can be added later.

Reply:
This would cause field duplication though...